### DISCUSSION TITLE
Option to show field short names on Publish screen

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#3340] [Feature Requests]
Currently there's no way for a DEV to know the short name of a field just by editing the entry - we have to edit the entry to find the name of the field, then go to  Fields and cross-reference.  Can we please get an option to add copyable text for field & grid-column short names to the regular Publish screens?

I'll let you decide how this should be controlled... IMO a user-specific toggle would be the best option, but a CP setting or even a config setting would be fine too.

**Comment by matthewjohns0n** [comment]
Seems like a good feature. I think it would probably be controlled with a CP setting on a per-member basis.

**Reply by pyrobob83** [reply]
![image](https://user-images.githubusercontent.com/8193519/234398976-94f38633-4632-4c89-be26-484b80eebdd3.png)

**Comment by intoeetive** [comment]
We're working on it in https://github.com/ExpressionEngine/ExpressionEngine/pull/3505

**Reply by intoeetive** [reply]
Getting this released in 7.4.0
